src/pages/subreddit/Subreddit.js

```diff
@@ -0,0 +1,55 @@
+import React, {useEffect, useState} from 'react';
+import './subreddit.css'
+import '../home/Home'
+import axios from "axios";
+import {useHistory, useParams} from "react-router-dom";
+
+function Subreddit (){
+
+    const [subInfo, setSubInfo] = useState([]);
+    const {subreddit} =useParams();
+    const history = useHistory();
+
+    function clickFunc(){
+        history.push('/')
```

Comment:
Als de button een échte "ga terug"-functie moet aanroepen, dan kun je hier ook de goBack functie aanroepen als het goed is:
```suggestion
        history.goBack();
```